test/ngSanitize/sanitizeSpec.js

```diff
@@ -314,6 +314,19 @@ describe('HTML', function() {
                    '<svg xmlns="http://www.w3.org/2000/svg"><a xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="https://example.com"></a></svg>');
     });
 
+    it('should sanitize SVG xml:base attribute values', function() {
+      expectHTML('<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><a xml:base="javascript:alert(1)//" href="#"></a></svg>')
```

Comment:
Just out of interest, do you know why the `//` is necessary?

Reply:
Yes, otherwise it is not valid JavaScript. E.g.

```
alert(1)#
```

compared to

```
alert(1)//#
```

(note the `//` makes the `#` into an inline comment